sycl/include/CL/sycl/types.hpp

```diff
@@ -589,7 +589,8 @@ template <typename Type, int NumElements> class vec {
   static constexpr size_t get_count() { return NumElements; }
   static constexpr size_t get_size() { return sizeof(m_Data); }
 
-  template <typename convertT, rounding_mode roundingMode>
+  template <typename convertT,
+            rounding_mode roundingMode = rounding_mode::automatic>
```

Comment:
I'd like that too but it's not spec. Propose to clarify whether it is possible to do so.

Reply:
In table 4.98 the spec says:
automatic - Default rounding mode for the SYCL vec class element type, rtz (round toward zero) for integer types and rte(roundtonearesteven)forﬂoating-point types. 
Is it not clear enough in the spec?

Reply:
Yes, that explains what automatic means for floating point types and for integer types. What behavior is expected.

But you change the API which is described in the specification.

I don't think it's a bad change. But if a user writes code with our SYCL, and uses what you wrote(cl::sycl::int4::template convert<float>()), then his code may not be compiled with another SYCL that follows the specification.

Reply:
That is a good point. Do you think asking for a spec update is valid for this?

Reply:
Yes, I agree with this change in SYCL specification. In OpenCL when converting by default, a rounding model is not written.